modules/images/dominant-color/load.php

```diff
@@ -0,0 +1,408 @@
+<?php
+/**
+ * Module Name: Dominant Color
+ * Description: Adds support to store dominant-color for an image and create a placeholder background with that color.
+ * Experimental: Yes
+ *
+ * @package performance-lab
+ * @since 1.0.0
```

Comment:
```suggestion
 * @since n.e.x.t
```

Also (I'm not going to leave separate comments on the rest of this PR), please add `@since n.e.x.t` to all new functions.